README.md

```diff
@@ -1 +1,128 @@
-# math-magicians
\ No newline at end of file
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Math Magicians\] ](#-math-magicians-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🤝 Contributing ](#-contributing-)
+  - [Show your support ](#show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Math Magicians] <a name="about-project"></a>
+
+> "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to: Make simple calculations and read a random math-related quote.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> I used the following tech stack to develop this project
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+- ReactJS
+- NPM
+
+In order to run this project you need:
+
+To have installed Nodejs in your Code editor. In case you don't have it, go to this [site](https://nodejs.org/en/) and download the setup based on your PC
+
+## Setup
+In a desired folder or directory in the CLI, run the command:
+
+```JavaScipt
+cd clone https://github.com/HanaSabih/math-magicians.git
+```
+
+### Install
+
+Navigate into the cloned folder or repository by running the command:
+```JavaScript
+ cd <folder/name of the repository>
+ ```
+Then run ```npm install``` to install all required dependencies (this will create a /node_modules folder).
+
+### Usage
+
+To run the project, execute the following command:
+
+Inside the newly created project, you can run some built-in commands:
+
+```npm start``` or ```yarn start```
+Runs the app in development mode.
+Open http://localhost:3000/ to view it in the browser.
+
+The page will automatically reload if you make changes to the code.
+You will see the build errors and lint warnings in the console.
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo]()
+
```

Comment:
[OPTIONAL]Since there is no live demo yet, I think it will be better if you put "Coming Soon" so that whoever is checking your code will know it's ongoing and not that you have a broken link.